### PR TITLE
[TACHYON-214] more Tachyon Shell commands for cluster monitoring

### PR DESCRIPTION
**Pull request by apc999**
Add a "du" comment to tfs so that one could check the size of a TFS file or folder

### CODE REVIEW

**Comment by haoyuan** [comment]
Jenkins, add to whitelist.

**Comment by haoyuan** [comment]
Jenkins, test this please.

File: core/src/main/java/tachyon/command/TFsShell.java [hunk 1]
```diff
@@ -808,4 +834,26 @@ private TachyonFS createFS(final TachyonURI path) throws IOException {
     TachyonFS tachyonFS = TachyonFS.get(new TachyonURI(qualifiedPath), mTachyonConf);
     return mCloser.register(tachyonFS);
   }
+
+  /**
+   * Calculates the size of a path (file or folder) specified by a TachyonURI.
+   *
+   * @param tachoinFS A TachyonFS
```
Comment (calvinjia) [inline core/src/main/java/tachyon/command/TFsShell.java]: tachyonFS

**Comment by calvinjia** [comment]
Aside from the typo, LGTM +1

**Pull request merged**